downstream_task_evaluation.py

```diff
@@ -646,7 +652,10 @@ def load_weights(
     model_dict.update(pretrained_dict)
 
     # 3. load the new state dict
-    model.load_state_dict(model_dict)
+    if multi_gpu_ft:
+        model.module.load_state_dict(model_dict)
+    else:
+        model.load_state_dict(model_dict)
```

Comment:
This is going to create bugs as not all the weights in the pre-training are available in the downstream because the heads are different.

Reply:
Do you mean these 4 lines specifically are going to create bugs, or the PR in general? Because for these 4 lines there's no change to the before behaviour, both `model.module.load_state_dict()` and `model.load_state_dict() `do the same thing. Just need to be sure that the 'module' prefix has been stripped first (which we do).